adding -march=armv4t to build libv8 on armel, and -Wno-psabi to fix gcc warnings on armel, see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=42748

--- a/SConstruct
+++ b/SConstruct
@@ -202,7 +202,8 @@
       },
       'unalignedaccesses:off' : {
         'CPPDEFINES' : ['CAN_USE_UNALIGNED_ACCESSES=0']
-      }
+      },
+      'CCFLAGS':      ['-mno-thumb-interwork', '-fno-tree-sink', '-Wno-psabi']
     },
     'simulator:arm': {
       'CCFLAGS':      ['-m32'],
